dags/process_elife_zip_dag.py

```diff
@@ -294,9 +294,11 @@ def send_article_to_content_service(upstream_task_id: str = None, **context) ->
 
     try:
         response.raise_for_status()
-    except HTTPError as e:
-        logger.error('%s HTTP Error= %s', response.status_code, response.text)
-        raise e
+    except HTTPError:
+        response_xml = etree.parse(BytesIO(response.content))
+        details = response_xml.find('{urn:ietf:rfc:7807}details')
+        logger.error('HTTPError: %s - %s', response.status_code, details.text)
```

Comment:
I agree with the logging level, any other field from the XML that would be useful to log? According to the RFC there is `title` and `type`

Reply:
I saw the test example, `title` only then

Reply:
have a look at the test code for an example of the response structure. Also remember that I still raise the original exception which will provide details about the response.